src/SignalR/samples/SignalRSamples/Hubs/Streaming.cs

```diff
@@ -12,16 +12,16 @@ namespace SignalRSamples.Hubs
 {
     public class Streaming : Hub
     {
-        public async IAsyncEnumerable<int> AsyncEnumerableCounter(int count, int delay)
+        public async IAsyncEnumerable<int> AsyncEnumerableCounter(int count, double delay)
```

Comment:
Presumably, this is an unrelated change to the json port, correct?

Reply:
No, System.Text.Json doesn't support casting double to int automatically.

Reply:
Is there a skipped test for this?

Reply:
No